sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -5490,6 +5377,43 @@ SUBROUTINE CLDRAD
            datapd(1:iend-ista+1,1:jend-jsta+1,cfld)=GRID1(ista:iend,jsta:jend)
          endif
       ENDIF
+#if 0
```

Comment:
@zhanglikate May I ask for the intention of the line "#if 0" and "endif"?

Reply:
It means that the parts between  "#if 0" and "#endif" are commented out.

Reply:
@zhanglikate I saw the warning message from compiling as:
```
/home/Wen.Meng/stmp2/kate/UPP/sorc/ncep_post.fd/CLDRAD.f(5380): warning #5117: Bad # preprocessor line
#if 0
-^
/home/Wen.Meng/stmp2/kate/UPP/sorc/ncep_post.fd/CLDRAD.f(5415): warning #5117: Bad # preprocessor line
#endif
```
The block is new code for outputting two new variables. You commented it out  temporarily?